install/terraform/modules/gke/cluster.tf

```diff
@@ -19,37 +19,49 @@ terraform {
 
 data "google_client_config" "default" {}
 
+# A list of all parameters used in interpolation var.cluster
+# Set values to default if not key was not set in original map
+locals {
```

Comment:
@chrisst you are our resident TF expert. This look good to you? If so, I'm happy to approve this PR :+1:

Reply:
Note that I have checked that removing `kubernetesVersion` from the `clluster` map
does not affect the result, as it was before.
https://github.com/aLekSer/agones/blob/terraform%2Fupdate-gke-version/build/main.tf#L26-L34

Reply:
LGTM!